tomviz/pvextensions/TomVizExtensions.xml

```diff
@@ -0,0 +1,149 @@
+<ServerManagerConfiguration>
```

Comment:
Please don't capitalize the V, we decided that tomviz is all lower case, it is acceptable to capitalize the first letter if following some convention, but the V should remain lower case.